### PR TITLE
Bug fix 1378, 1477, 1489, 1490, 1486, 1479, 1488, 1469, 1466

### PR DESCRIPTION
**Pull request by brebaroluk**
changed the logic for managing popular articles in the homepage. it is not storing the posts in the postStore now

### CODE REVIEW

**Comment by MITCHELLKURTZMAN** [comment]
Looks good, will approve after small merge conflicts are solved. 

Also some of the files have been removed and re-added, I'm guessing this was a permissions issue?

**Comment by brebaroluk** [comment]
Yes, removed/re-added files were a permissions issue. Will check out the merge conflicts.

**Review by PeeEmKay — APPROVED** [review]

**Pull request merged**
